node-js-basics-101/node_modules/xmlhttprequest-ssl/tests/test-events.js

```diff
@@ -0,0 +1,50 @@
+var sys = require("util")
```

Comment:
__[jshint]__
* Misleading line break before ','; readers may interpret this as an expression boundary.

<!-- Comment by [SideCI](https://sideci.com) -->